scan-framework/src/main/java/com/getbouncer/scan/framework/ml/TensorFlowLiteAnalyzer.kt

```diff
@@ -80,4 +90,4 @@ abstract class TFLAnalyzerFactory<Output : Analyzer<*, *, *>>(private val loader
         }
         loadedModel
     }
-}
+}
```

Comment:
Please make sure to keep the new lines at the end of the files.

Reply:
Will do, I've set up run configs for the linter now, so ktlint can catch my errors before I push.